java/client/build.gradle

```diff
@@ -234,3 +235,22 @@ jar {
     // placeholder will be renamed by platform+arch on the release workflow java-cd.yml
     archiveClassifier = "placeholder"
 }
+
+spotbugsMain {
+    reports {
+        html {
+            required = true
+            outputLocation = file("$buildDir/reports/spotbugs/main/spotbugs.html")
```

Comment:
Can you update CI workflow to upload reports please?

Reply:
Added - Spotbugs are now saved for each CI build:
reference: https://github.com/aws/glide-for-redis/actions/runs/9734180206?pr=1719